fuzz/fuzz.py

```diff
@@ -0,0 +1,460 @@
+import asyncio
```

Comment:
I don't think this file should have comeback?

Reply:
Indeed, probably just needs a rebase

Reply:
deleted fuzz